ratatui/src/backend.rs

```diff
@@ -109,20 +109,20 @@ use crate::{
     layout::{Position, Size},
 };
 
-#[cfg(all(not(windows), feature = "termion"))]
-mod termion;
-#[cfg(all(not(windows), feature = "termion"))]
-pub use self::termion::TermionBackend;
-
 #[cfg(feature = "crossterm")]
 mod crossterm;
 #[cfg(feature = "crossterm")]
-pub use self::crossterm::CrosstermBackend;
+pub use self::crossterm::{CrosstermBackend, FromCrossterm, IntoCrossterm};
```

Comment:
We should also update the `prelude` for these new traits.

Reply:
b7e749a99..25baa29ee